release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1293,6 +1293,21 @@ In the following tables, features are marked with the following statuses:
 |Dual-stack networking with installer-provisioned infrastructure
 |Not Available
 |Technology Preview
+|General Availability
+
+|Dual-stack networking with user-provisioned infrastructure
+|Not Available
+|Technology Preview
```

Comment:
UPI was not available as TP in 4.14 only IPI was.
UPI work was developed on 4.15 and is moving straight to GA on 4.15. Can you update that? thanks

Reply:
Updated - thanks @MaysaMacedo !